addons/hw_drivers/controllers/driver.py

```diff
@@ -316,8 +320,9 @@ def video_loop(self):
                 dev = v4l2.v4l2_capability()
                 ioctl(path, v4l2.VIDIOC_QUERYCAP, dev)
                 dev.interface = video
+                dev.identifier = dev.bus_info.decode('utf-8')
                 iot_device = IoTDevice(dev, 'video')
-                camera_devices[dev.bus_info.decode('utf-8')] = iot_device
+                camera_devices[dev.identifier] = iot_device
```

Comment:
you don't have to increment here?

Reply:
video interface have already a unique identifier by raspberry